src/com/owncloud/android/ui/fragment/SearchShareesFragment.java

```diff
@@ -139,6 +141,8 @@ public boolean onQueryTextChange(String newText) {
             }
         });
 
+        view.setMinimumWidth(R.dimen.share_user_groups_width);
```

Comment:
This dimension was being hardcoded in `res/layout/search_users_groups_layout.xml `before this change and I created a new dimension (`share_user_groups_width`).  Apart from that, the declaration can be moved back to the layout instead of setting it from the Fragment.